music21/metadata/__init__.py

```diff
@@ -546,26 +546,15 @@ def contributors(self) -> t.List[Contributor]:
         Returns [] if no Contributors exist.
 
         >>> md = metadata.Metadata()
-        >>> md['composer'] = ['Ludwig', 'Wolfgang']
-        >>> md['librettist'] = 'Joe'
-        >>> md.add('architect', 'John')      # 'architect' is new in 2022
-        >>> md.add('title', 'Caveat Emptor') # not a contributor
+        >>> md['composer'] = ['Richard Strauss']
+        >>> md.librettist = 'Oscar Wilde'
+        >>> md.add('title', 'Salome') # not a contributor
         >>> contribs = md.contributors
         >>> contribs
-        [<music21.metadata.primitives.Contributor composer:Ludwig>,
-        <music21.metadata.primitives.Contributor composer:Wolfgang>,
-        <music21.metadata.primitives.Contributor librettist:Joe>,
-        <music21.metadata.primitives.Contributor architect:John>]
+        [<music21.metadata.primitives.Contributor composer:Richard Strauss>,
+        <music21.metadata.primitives.Contributor librettist:Oscar Wilde>]
```

Comment:
add one space to visually align and show that Oscar Wilde is in the bracket.

Reply:
Though if this is a generated-on-the-fly set whose appending does not affect the metadata, much better to make it a tuple, so that no one is surprised by doing `md.contributors.append('xxx')`

Reply:
Indented, and it's a tuple.